Demo/src/main/java/com/braintreepayments/demo/PayPalMessagingFragment.kt

```diff
@@ -22,6 +24,7 @@ class PayPalMessagingFragment : BaseFragment(), PayPalMessagingListener {
         return inflater.inflate(R.layout.fragment_paypal_messaging, container, false)
     }
 
+    @OptIn(ExperimentalBetaApi::class)
```

Comment:
Just curious - is this required if we've already opted the entire fragment class in?

Reply:
Looks like it's not required. Removing it.